Whisky/Views/ContentView.swift

```diff
@@ -15,23 +15,37 @@ struct ContentView: View {
 
     var body: some View {
         NavigationSplitView {
-            List(selection: $selected) {
-                ForEach(bottleVM.bottles, id: \.url) { bottle in
-                    BottleListEntry(bottle: bottle, selected: $selected)
-                }
-                ForEach(bottleVM.inFlightBottles, id: \.self) { inFlight in
-                    HStack {
-                        Text(inFlight)
-                            .foregroundStyle(.secondary)
-                        Spacer()
-                        ProgressView()
-                            .controlSize(.small)
+            VStack {
```

Comment:
Don't add text to the sidebar. It is not common practice on macOS

Reply:
How can we communicate this in a macos way?

Reply:
The button in the main view is sufficient